sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -76,50 +75,69 @@ export interface ChatClientOptions extends PipelineOptions {
 }
 
 // @public
-export interface ChatMessage extends Omit<RestChatMessage, "senderId" | "content"> {
+export interface ChatMessage {
     content?: ChatMessageContent;
-    sender?: CommunicationUserIdentifier;
+    createdOn: Date;
+    deletedOn?: Date;
+    editedOn?: Date;
+    id: string;
+    senderCommunicationIdentifier?: CommunicationIdentifierModel;
+    senderDisplayName?: string;
+    sequenceId: string;
+    type: ChatMessageType;
+    version: string;
 }
 
-// @public (undocumented)
-export interface ChatMessageContent extends Omit<RestChatMessageContent, "participants"> {
+// @public
+export interface ChatMessageContent {
+    initiatorCommunicationIdentifier?: CommunicationIdentifierModel;
+    message?: string;
     participants?: ChatParticipant[];
+    topic?: string;
 }
 
 // @public
-export interface ChatMessageReadReceipt extends Omit<RestChatMessageReadReceipt, "senderId"> {
-    readonly sender: CommunicationUserIdentifier;
+export interface ChatMessageReadReceipt {
+    chatMessageId: string;
+    readOn: Date;
+    senderCommunicationIdentifier: CommunicationIdentifierModel;
 }
 
 // @public
 export type ChatMessageType = string;
 
 // @public
-export interface ChatParticipant extends Omit<RestChatParticipant, "id"> {
-    user: CommunicationUserIdentifier;
+export interface ChatParticipant {
+    communicationIdentifier: CommunicationIdentifierModel;
+    displayName?: string;
+    shareHistoryTime?: Date;
 }
 
 // @public
-export interface ChatThread extends Omit<RestChatThread, "createdBy"> {
-    readonly createdBy?: CommunicationUserIdentifier;
+export interface ChatThread {
+    createdByCommunicationIdentifier: CommunicationIdentifierModel;
```

Comment:
```suggestion
    createdBy: CommunicationIdentifier;
```
We call it `initiator` in that other `ChatMessageContent` type (but that one seems weird)